bin/makefile-tail

```diff
@@ -183,108 +183,142 @@ $(OBJECTDIR)vdate.o: $(LIBFILES) $(EXTFILES) $(OSARCHDIR)mkvdate
 
-$(OBJECTDIR)array3.o :  $(SRCDIR)array3.c  $(REQUIRED-INCS) $(INCDIR)lispemul.h  \
-	 $(INCDIR)lspglob.h  $(INCDIR)adr68k.h \
-	 $(INCDIR)lispmap.h  $(INCDIR)lsptypes.h  $(INCDIR)emlglob.h \
-	 $(INCDIR)arith.h $(INCDIR)medleyfp.h  $(INCDIR)my.h
+$(OBJECTDIR)array3.o: $(SRCDIR)array3.c $(REQUIRED-INCS) \
+  $(INCDIR)lispemul.h $(INCDIR)lspglob.h \
+  $(INCDIR)ifpage.h $(INCDIR)iopage.h $(INCDIR)miscstat.h $(INCDIR)adr68k.h \
+  $(INCDIR)lispmap.h $(INCDIR)lsptypes.h $(INCDIR)emlglob.h \
+  $(INCDIR)array3defs.h $(INCDIR)mkcelldefs.h $(INCDIR)arith.h $(INCDIR)my.h
 	$(CC) $(RFLAGS) $(SRCDIR)array3.c -o $(OBJECTDIR)array3.o
 
-$(OBJECTDIR)array5.o :  $(SRCDIR)array5.c  $(REQUIRED-INCS) $(INCDIR)lispemul.h  \
-	 $(INCDIR)lspglob.h  $(INCDIR)adr68k.h \
-	 $(INCDIR)lispmap.h  $(INCDIR)lsptypes.h  $(INCDIR)emlglob.h \
-	 $(INCDIR)arith.h $(INCDIR)medleyfp.h $(INCDIR)my.h
+$(OBJECTDIR)array5.o: $(SRCDIR)array5.c $(REQUIRED-INCS) \
+  $(INCDIR)lispemul.h $(INCDIR)lspglob.h \
+  $(INCDIR)ifpage.h $(INCDIR)iopage.h $(INCDIR)miscstat.h $(INCDIR)adr68k.h \
+  $(INCDIR)lispmap.h $(INCDIR)lsptypes.h $(INCDIR)emlglob.h \
+  $(INCDIR)array5defs.h $(INCDIR)mkcelldefs.h $(INCDIR)arith.h $(INCDIR)my.h
 	$(CC) $(RFLAGS) $(SRCDIR)array5.c -o $(OBJECTDIR)array5.o
 
-$(OBJECTDIR)bin.o :  $(SRCDIR)bin.c  $(REQUIRED-INCS) $(INCDIR)lispemul.h  \
-	 $(INCDIR)lispmap.h  $(INCDIR)emlglob.h \
-	 $(INCDIR)lspglob.h  $(INCDIR)lsptypes.h  $(INCDIR)address.h \
-	 $(INCDIR)adr68k.h  $(INCDIR)cell.h  $(INCDIR)stream.h
+$(OBJECTDIR)bin.o: $(SRCDIR)bin.c $(REQUIRED-INCS) \
+  $(INCDIR)lispemul.h $(INCDIR)lispmap.h \
+  $(INCDIR)emlglob.h $(INCDIR)lspglob.h $(INCDIR)ifpage.h $(INCDIR)iopage.h \
+  $(INCDIR)miscstat.h $(INCDIR)lsptypes.h $(INCDIR)address.h $(INCDIR)adr68k.h \
+  $(INCDIR)cell.h $(INCDIR)stream.h $(INCDIR)bindefs.h
 	$(CC) $(RFLAGS) $(SRCDIR)bin.c -o $(OBJECTDIR)bin.o
 
-$(OBJECTDIR)binds.o :  $(SRCDIR)binds.c  $(REQUIRED-INCS) $(INCDIR)lispemul.h  \
-	 $(INCDIR)lspglob.h  $(INCDIR)emlglob.h
+$(OBJECTDIR)binds.o: $(SRCDIR)binds.c $(REQUIRED-INCS) \
+  $(INCDIR)lispemul.h $(INCDIR)lspglob.h \
+  $(INCDIR)ifpage.h $(INCDIR)iopage.h $(INCDIR)miscstat.h $(INCDIR)emlglob.h \
+  $(INCDIR)bindsdefs.h
 	$(CC) $(RFLAGS) $(SRCDIR)binds.c -o $(OBJECTDIR)binds.o
 
-$(OBJECTDIR)bitblt.o :  $(SRCDIR)bitblt.c  $(REQUIRED-INCS) $(INCDIR)lispemul.h \
-	  $(INCDIR)lspglob.h  $(INCDIR)lispmap.h \
-	 $(INCDIR)emlglob.h  $(INCDIR)adr68k.h  $(INCDIR)address.h \
-	 $(INCDIR)pilotbbt.h  $(INCDIR)display.h  $(INCDIR)bitblt.h \
-	 $(INCDIR)bb.h
+$(OBJECTDIR)bitblt.o: $(SRCDIR)bitblt.c $(REQUIRED-INCS) \
+  $(INCDIR)lispemul.h $(INCDIR)lspglob.h \
+  $(INCDIR)ifpage.h $(INCDIR)iopage.h $(INCDIR)miscstat.h $(INCDIR)lispmap.h \
+  $(INCDIR)adr68k.h $(INCDIR)address.h $(INCDIR)pilotbbt.h $(INCDIR)display.h \
+  $(INCDIR)bitblt.h $(INCDIR)bb.h $(INCDIR)bitbltdefs.h $(INCDIR)initdspdefs.h
 	$(CC) $(RFLAGS) $(SRCDIR)bitblt.c -o $(OBJECTDIR)bitblt.o
```

Comment:
Change in leading white space here...

Reply:
Every one that I updated went from <tab><space> to <space><space> -- I didn't check to see if the list of dependencies changed, so if it was previously correct it will show up as just a white-space change.  I'm not a fan of using tabs in the makefile except where they're necessary.  I'm also OK with the ones that I've based on the compiler generated list looking different from the ones that are either unused, or unverified (e.g., the lpXXX.o that I haven't compiled).